ci/github-actions.md

```diff
@@ -0,0 +1,125 @@
+# GitHub Actions
+
+This guide provides documentation for setting up continuous integration with [GitHub Actions](https://help.github.com/en/actions) for different types of projects.
+
+## Contents
+
+- [Running containerized tests](#running-containerized-tests)
+- [Deploying a Python package](#deploying-a-python-package)
+- [Resources](#resources)
+
+## Running containerized tests
+
+To initialize a workflow for running containerized tests, start by creating a new feature branch in your repo. Then, create a directory called `.github/workflows/` at the root of your repo. GitHub Actions will read any workflows from this directory and run them automatically.
+
+Next, define a file called `main.yml` in the workflows directory, and paste in the following configuration:
```

Comment:
Could you add a sentence about the expected behavior of this? Should it be running for any commit, or just to master?

Reply:
Added this line below the code block:

> Due to the `on` block, this workflow will run the `test` job on all commits to `master` and all commits to pull requests that have been opened against `master`.